exporters/trace/stackdriver/src/test/java/io/opencensus/exporter/trace/stackdriver/StackdriverV2ExporterHandlerProtoTest.java

```diff
@@ -258,6 +271,39 @@ public void generateSpan() {
         .isEqualTo(Int32Value.newBuilder().setValue(CHILD_SPAN_COUNT).build());
   }
 
+  @Test
+  public void generateSpan_WithResourceLabels() {
```

Comment:
This change looks good, though I think we should also test the other two ResourceTypes, as in 57765e8cea76cf612fda1251a8f99bba750c12ff.

Reply:
Done. All these tests tested the same code path though, I'll update this to test parsing MonitoredResource instead, after https://github.com/census-instrumentation/opencensus-java/pull/1250 is merged.